sys-devel/ucc-1.1-r0: QA Notice: make jobserver unavailable

```
emerge -1 ucc --quiet-build n
Calculating dependencies... done!

>>> Verifying ebuild manifests

>>> Emerging (1 of 1) sys-devel/ucc-1.1-r0::local
 * ucc160.zip BLAKE2B SHA512 size ;-) ...                                                                                [ ok ]
>>> Unpacking source...
>>> Unpacking ucc160.zip to /var/tmp/portage/sys-devel/ucc-1.1-r0/work
>>> Source unpacked in /var/tmp/portage/sys-devel/ucc-1.1-r0/work
>>> Preparing source in /var/tmp/portage/sys-devel/ucc-1.1-r0/work/ucc ...
 * Applying fix-ignore-cflags-and-ldflags.patch ...                                                                      [ ok ]
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/sys-devel/ucc-1.1-r0/work/ucc ...
>>> Source configured.
>>> Compiling source in /var/tmp/portage/sys-devel/ucc-1.1-r0/work/ucc ...
make -j4 
make -C driver
make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
make[1]: Entering directory '/var/tmp/portage/sys-devel/ucc-1.1-r0/work/ucc/driver'
cc -o ucc -march=ivybridge -O3 -pipe -Wl,-O2 -Wl,--as-needed ucc.c linux.c
ucc.c: In function ‘AddFile’:
ucc.c:183:13: warning: implicit declaration of function ‘access’ [-Wimplicit-function-declaration]
  183 |         if (access(filename, 4) != 0)
      |             ^~~~~~
ucc.c: In function ‘ShowHelp’:
ucc.c:289:17: warning: format not a string literal and no format arguments [-Wformat-security]
  289 |                 printf(msg[i]);
      |                 ^~~~~~
linux.c: In function ‘Execute’:
linux.c:46:21: warning: implicit declaration of function ‘wait’ [-Wimplicit-function-declaration]
   46 |         while ((n = wait(&status)) != pid && n != -1)
      |                     ^~~~
make[1]: Leaving directory '/var/tmp/portage/sys-devel/ucc-1.1-r0/work/ucc/driver'
make -C ucl
make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
make[1]: Entering directory '/var/tmp/portage/sys-devel/ucc-1.1-r0/work/ucc/ucl'
cc -march=ivybridge -O3 -pipe   -c -o alloc.o alloc.c
cc -march=ivybridge -O3 -pipe   -c -o ast.o ast.c
cc -march=ivybridge -O3 -pipe   -c -o decl.o decl.c
cc -march=ivybridge -O3 -pipe   -c -o declchk.o declchk.c
cc -march=ivybridge -O3 -pipe   -c -o dumpast.o dumpast.c
dumpast.c: In function ‘DumpExpression’:
dumpast.c:105:17: warning: format not a string literal and no format arguments [-Wformat-security]
  105 |                 fprintf(ASTFile, TypeToString(expr->ty));
      |                 ^~~~~~~
cc -march=ivybridge -O3 -pipe   -c -o emit.o emit.c
cc -march=ivybridge -O3 -pipe   -c -o error.o error.c
cc -march=ivybridge -O3 -pipe   -c -o expr.o expr.c
cc -march=ivybridge -O3 -pipe   -c -o exprchk.o exprchk.c
cc -march=ivybridge -O3 -pipe   -c -o flow.o flow.c
cc -march=ivybridge -O3 -pipe   -c -o fold.o fold.c
cc -march=ivybridge -O3 -pipe   -c -o gen.o gen.c
gen.c: In function ‘TryAddValue’:
gen.c:292:18: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  292 |         int h = ((unsigned)src1 + (unsigned)src2 + op) & 15;
      |                  ^
gen.c:292:35: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  292 |         int h = ((unsigned)src1 + (unsigned)src2 + op) & 15;
      |                                   ^
cc -march=ivybridge -O3 -pipe   -c -o input.o input.c
input.c: In function ‘ReadSourceFile’:
input.c:102:22: warning: cast to pointer from integer of different size [-Wint-to-pointer-cast]
  102 |         Input.file = (void *)fno;
      |                      ^
input.c: In function ‘CloseSourceFile’:
input.c:129:15: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  129 |         close((int)Input.file);
      |               ^
cc -march=ivybridge -O3 -pipe   -c -o lex.o lex.c
cc -march=ivybridge -O3 -pipe   -c -o output.o output.c
cc -march=ivybridge -O3 -pipe   -c -o reg.o reg.c
cc -march=ivybridge -O3 -pipe   -c -o simp.o simp.c
cc -march=ivybridge -O3 -pipe   -c -o stmt.o stmt.c
cc -march=ivybridge -O3 -pipe   -c -o stmtchk.o stmtchk.c
cc -march=ivybridge -O3 -pipe   -c -o str.o str.c
cc -march=ivybridge -O3 -pipe   -c -o symbol.o symbol.c
cc -march=ivybridge -O3 -pipe   -c -o tranexpr.o tranexpr.c
cc -march=ivybridge -O3 -pipe   -c -o transtmt.o transtmt.c
cc -march=ivybridge -O3 -pipe   -c -o type.o type.c
cc -march=ivybridge -O3 -pipe   -c -o ucl.o ucl.c
cc -march=ivybridge -O3 -pipe   -c -o uildasm.o uildasm.c
cc -march=ivybridge -O3 -pipe   -c -o vector.o vector.c
cc -march=ivybridge -O3 -pipe   -c -o x86.o x86.c
cc -march=ivybridge -O3 -pipe   -c -o x86linux.o x86linux.c
cc -march=ivybridge -O3 -pipe   -c -o assert.o assert.c
cc -o ucl -march=ivybridge -O3 -pipe -Wl,-O2 -Wl,--as-needed alloc.o ast.o decl.o declchk.o dumpast.o emit.o error.o expr.o exprchk.o flow.o fold.o gen.o input.o lex.o output.o reg.o simp.o stmt.o stmtchk.o str.o symbol.o tranexpr.o transtmt.o type.o ucl.o uildasm.o vector.o x86.o x86linux.o
make[1]: Leaving directory '/var/tmp/portage/sys-devel/ucc-1.1-r0/work/ucc/ucl'
>>> Source compiled.
>>> Test phase [not enabled]: sys-devel/ucc-1.1-r0

>>> Install sys-devel/ucc-1.1-r0 into /var/tmp/portage/sys-devel/ucc-1.1-r0/image
>>> Completed installing sys-devel/ucc-1.1-r0 into /var/tmp/portage/sys-devel/ucc-1.1-r0/image

 * Final size of build directory: 1488 KiB (1.4 MiB)
 * Final size of installed tree:   244 KiB

 * QA Notice: make jobserver unavailable:
 * 
 * 	make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
 * 	make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
strip: x86_64-pc-linux-gnu-strip --strip-unneeded -N __gentoo_check_ldflags__ -R .comment -R .GCC.command.line -R .note.gnu.gold-version
   /usr/lib/ucc/assert.o
   /usr/bin/ucc
   /usr/bin/ucl

>>> Installing (1 of 1) sys-devel/ucc-1.1-r0::local
>>> Auto-cleaning packages...

>>> No outdated packages were found on your system.

 * GNU info directory index is up-to-date.
```

--- a/sys-devel/ucc/ucc-1.1-r0.ebuild
+++ b/sys-devel/ucc/ucc-1.1-r0.ebuild
@@ -14,7 +14,7 @@ LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-RESTRICT="mirror"
+RESTRICT="mirror test"
 IUSE=""
 
 DEPEND=""
@@ -24,8 +24,9 @@ S="${WORKDIR}/${PN}"
 
 src_prepare() {
 	default
-	sed -i -e 's:/usr/local/lib/ucc/:/usr/lib/ucc/:' "${S}/driver/linux.c"
-	sed -i -e 's:UCCDIR \"ucl\":\"/usr/bin/ucl\":' "${S}/driver/linux.c"
+	sed -i 's:/usr/local/lib/ucc/:/usr/lib/ucc/:' "${S}/driver/linux.c" || die
+	sed -i 's:UCCDIR \"ucl\":\"/usr/bin/ucl\":' "${S}/driver/linux.c" || die
+	sed -i 's:make -C:$(MAKE) -C:' "${S}/Makefile" || die
 	eapply "${FILESDIR}/fix-ignore-cflags-and-ldflags.patch"
 }
 

